lib/internal/assert/assertion_error.js

```diff
@@ -319,16 +319,15 @@ class AssertionError extends Error {
       actual,
       expected
     } = options;
-
     const limit = Error.stackTraceLimit;
     Error.stackTraceLimit = 0;
 
     if (message != null) {
       super(String(message));
     } else {
       if (process.stderr.isTTY) {
-        // Reset on each call to make sure we handle dynamically set environment
-        // variables correct.
+        // Reset on each call to make sure we handle dynamically set
+        // environment variables correct.
```

Comment:
Why are there changes that are just line breaks? I thought maybe you used the different js linter

Reply:
and there are many unnecessary changes like I said above